app/pages/lab/iiif/components/UploadButton.jsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router';
+
+import { createSubjectSet } from './helpers';
+import { useSubjectUploads } from './hooks'
+
+function subjectSetSnapshot(manifest, metadata, project) {
+  return {
+    display_name: manifest.label,
+    metadata,
+    links:
+      { project: project.id }
+  }
+}
+
+function subjectSnapshot(metadata, project, subject) {
+  const { priority, ...subjectMetadata } = subject.metadata;
+  const { locations } = subject;
+  return {
+    locations,
+    metadata: {
+      ['#priority']: priority,
```

Comment:
[minor][not blocking in any way] Out of curiosity, why is the key name here `['#priority']` instead of just `'#priority'`? I understand that the brackets around `[key_name]` is for [computed property names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#computed_property_names)) but I'm not aware if the '#priority' string changes in this context?

Reply:
I think this is left over from an earlier version of the code.